static_report/src/components/SingleReport/SRTableList/SRTableListColumnLabel.tsx

```diff
@@ -0,0 +1,23 @@
+import { Flex, Center, Text, Icon } from '@chakra-ui/react';
+import { FiGrid } from 'react-icons/fi';
+
+export function SRTableListColumnLabel({
+  name,
+  visibleIcon = true,
```

Comment:
`hasIcon` to designate boolean by name?
`visible` makes me think about  CSS's `visibility` property, which is not the case here

Reply:
This prop is redundant, I will remove it.